src/gallery/Gallery.js

```diff
@@ -12,18 +13,39 @@ export const Gallery = {
    * @returns {void}
   */
   init() {
-    this.galleryEls = document.querySelectorAll(`.js-gallery`);
-    this.setupGalleries();
+    const galleryEls = document.querySelectorAll(`.js-gallery`);
+    this.setupGalleries(galleryEls);
+    this.setupImageGalleries();
   },
 
   /**
    * Instantiates Flickity on each gallery, with their respective options
+   * @param {object} Gallery DOM elements to setup
    * @returns {void}
   */
-  setupGalleries() {
-    [...this.galleryEls].forEach((el) => {
+  setupGalleries(galleryEls) {
+    [...galleryEls].forEach((el) => {
       const options = JSON.parse(el.getAttribute(`data-options`));
       new Flickity(el, options);
     });
   },
+
+  /**
+   * Sets custom imgix settings for gallery images, instantiating Flickity and
+   * fading in the gallery once the first image has loaded
+   * @returns {void}
+  */
+  setupImageGalleries() {
```

Comment:
This is great! addresses this card that's been in the icebox :D https://trello.com/c/MnLS6IIJ